src/vsr/replica.zig

```diff
@@ -4289,6 +4306,29 @@ pub fn ReplicaType(
             return self.commit_dispatch_resume();
         }
 
+        fn commit_checkpoint_durable(self: *Replica) enum { ready, pending } {
+            assert(self.commit_stage == .checkpoint_durable);
+
+            if (!self.grid.free_set.checkpoint_durable and
+                vsr.Checkpoint.durable(self.op_checkpoint(), self.commit_min))
+            {
```

Comment:
Nit, but now that the `else` branch is very simple, we can avoid using a compound condition by rewriting it to:

```suggestion
            if (self.grid.free_set.checkpoint_durable) return .ready;
            if (!vsr.Checkpoint.durable(self.op_checkpoint(), self.commit_min)) return .ready;
            
            ...
```

Reply:
I really like this nit! Makes the code more beautiful. :)